lib/cc/engine/analyzers/violation.rb

```diff
@@ -45,7 +64,9 @@ def sorted_hashes
         end
 
         def other_sexps
-          @other_locations ||= sorted_hashes.drop(1)
+          sorted = sorted_hashes
```

Comment:
But... [we get the location *from* the sexp](https://github.com/codeclimate/codeclimate-duplication/blob/abh-python-tuning/lib/cc/engine/analyzers/violation.rb#L84-L86). Is it just not taken into account by `Sexp#==`?